code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -22,6 +22,7 @@
 	var/elimination = 0
 	anchored = TRUE
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
+	mouse_opacity = 2 // Easier to click on in melee, they're giant targets anyway
```

Comment:
The collision box is the sprite itself, which scales with the transform, therefore they shouldn't be 16x16 sized icons with guge 256x256 bounding boxes, like I believe you're worried about Joan.